docs/openapi.json

```diff
@@ -13,25 +13,46 @@
     "version" : "1.0.0"
   },
   "paths" : {
-    "/contracts/v1/version" : {
-      "get" : {
-        "tags" : [ "version" ],
-        "summary" : "callable way of retrieving current version",
-        "description" : "callable way (compared to analyzing the URL of the endpoint) of retrieving current version. As well good for testing purposes as operation is get only",
-        "operationId" : "version",
+    "/contracts/v1/documents" : {
+      "post" : {
+        "tags" : [ "documents" ],
+        "summary" : "Upload a document for a contract.",
+        "description" : "Please note that this operation needs to be called per document for a contract to be uploaded. The response contains a handle to the document. This handle should be provided with the contract to be uploaded via uploadContract",
+        "operationId" : "uploadDocument",
+        "requestBody" : {
+          "description" : "a documnent that is part of a contract",
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/Document"
+              }
+            }
+          },
+          "required" : true
+        },
         "responses" : {
           "default" : {
-            "description" : "version identifier",
+            "description" : "A handle to the document for providing with a new contract to be uploaded via POST contracts",
             "content" : {
               "application/json" : {
                 "schema" : {
-                  "$ref" : "#/components/schemas/Version"
+                  "$ref" : "#/components/schemas/FileHandle"
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "Invalid Document is provided. See Error details for more information about validation issues",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/ErrorResponse"
                 }
               }
             }
           },
           "503" : {
-            "description" : "technical issue on our side, please retry later",
+            "description" : "Technical issue on our side, please retry later",
```

Comment:
I think it makes sense to be neutral in naming => "our"
Would rephrase to "Technical issue on server side"

Reply:
ok - done